doc/api/inspector.md

```diff
@@ -0,0 +1,80 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: REPLACEME
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
+
+It needs to be connected before the messages can be sent to the inspector
+backend.
```

Comment:
The constructor needs docs, even though it doesn't take args, without I had to guess that `session = new inspector.Session()`